src/atwho.js

```diff
@@ -56,7 +52,7 @@ define([
                     return choice.name && !_.isUndefined(choice.displayLabel);
                 })
                 .value();
-    }, 500);
+    }, 0);
```

Comment:
Ah. that slipped in when I was debugging tests. https://github.com/dimagi/Vellum/pull/583